src/file-uploader/file-uploader.component.ts

```diff
@@ -229,31 +229,29 @@ export class FileUploader {
 
 		const transferredFiles = Array.from(event.dataTransfer.files);
 
-		if (!this.accept.length) {
-			transferredFiles.forEach(file => {
-				const fileItem = this.createFileItem(file);
-				this.files.add(fileItem);
-				this.filesChange.emit(this.files);
-			});
-
-			this.value = this.files;
-			this.dragOver = false;
-			return;
-		}
-
 		transferredFiles.filter(({ name, type }) => {
 			// Get the file extension and add a "." to the beginning.
 			const fileExtension = name.split(".").pop().replace(/^/, ".");
+			if (this.accept.length === 0) {
+				return true;
+			}
 			// Check if the accept array contains the mime type or extension of the file.
 			return this.accept.includes(type) || this.accept.includes(fileExtension);
 		}).forEach(file => {
-			const fileItem = this.createFileItem(file);
-			this.files.add(fileItem);
-			this.filesChange.emit(this.files);
+
+			if ((this.files.size < 1) || this.multiple) {
+				const fileItem = this.createFileItem(file);
+
```

Comment:
There's no need to space everything out like this. All three lines essentially work together to accomplish a joint thing, they are contextually related, which is why they were grouped in the first place.
Readability doesn't always mean adding more space :)
```suggestion
```